pom.xml

```diff
@@ -33,12 +33,12 @@
     </scm>
 
     <properties>
-        <api_package>javax.websocket</api_package>
+        <api_package>jakarta.websocket</api_package>
         <spec_version>1.1</spec_version>
         <!--<spec_build>02</spec_build>-->
         <new_spec_version>1.1</new_spec_version>
         <spec_impl_version>1.1.2</spec_impl_version>
```

Comment:
I need to remind myself exactly what these do but my initial expectation is that these version numbers also need to change to reflect the update to 2.0.0.

Reply:
Yup, those were missed.
Fixing.